spec/helpers/item_seasons_helper_spec.rb

```diff
@@ -3,6 +3,10 @@
 require 'rails_helper'
 
 RSpec.describe ItemSeasonsHelper, type: :helper do
+  let(:item_season_jan) { ItemSeason.create(produce_item_id: 1, month_index: 1, country_code: 'br') }
```

Comment:
**issue (not for this PR):** Seeing `produce_item_id` makes me wonder if a foreign key was created. I would expect this to fail on creation, yet it doesn't.

I've created an issue to address it: https://github.com/vasconsaurus/produce_in_season/issues/51

**question:** Any reason for not using factories here?

**suggestion:** Building the item (using `new` instead of `create`) wouldn't persist the item in the DB. This would made tests faster and is a good practice in the long run (as the codebase grows, time it takes to run the test suite becomes a problem)

Reply:
> question: Any reason for not using factories here?

Not really, it was easier to write that way 😅  I changed it to use a factory. WDYT?

> issue (not for this PR): Seeing produce_item_id makes me wonder if a foreign key was created. I would expect this to fail on creation, yet it doesn't.

Can you maybe explain this issue? Or could we talk about it later, so I understand it better?

Reply:
About foreign keys: in simplified terms they are links from one table to the other (in the database level). For instance, if you try to delete a produce item that has item seasons, PostgreSQL will complain saying that there are seasons attached to that item. As a alternative, you can also instruct the database software (RDMS – relational database management system) to do X or Y when they need to handle these cases.

I do recommend reading [this](https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-foreign-key/), which will provide you a better explanation on what foreign keys are and why use them :p

Reply:
I know what they are, but I will read the article. I was confused because I thought we were using them. Clearly, I must have set up something wrong 😓

Reply:
Aaah, sorry for that. I missed the foreign key reference [here](https://github.com/vasconsaurus/produce_in_season/blob/e93202d63b49813e24bc7d8d5971cb15c3126622/db/schema.rb#L34). My bad.

You shouldn't be able to create things with an arbitrary id, tho. I'll investigate this later.